config.yaml

```diff
@@ -38,15 +38,20 @@ params:
   mash:
     sketch_size: 1000
   kraken:
-    kraken_db: test_data/kraken_test_db
-    bracken_db: test_data/kraken_test_db/75mers_distr.txt
-    # kraken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken
-    # bracken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken/75mers_distr.txt
+    # db: test_data/kraken_test_db
+    # kmers: test_data/kraken_test_db/75mers_distr.txt
+    db: /home/qiz173/Works/WoL/profiling/ref107/kraken
+    kmers: /home/qiz173/Works/WoL/profiling/ref107/kraken/75mers_distr.txt
+    threads: 8
+  shogun:
+    db: /projects/ag500/shogun/db/rep82
+    threads: 8
```

Comment:
Resource specification happens in a separate file, cluster.json... Haven't integrated that yet.

Reply:
Got it! What shall I do now?

Reply:
Take a look at the other rules files. You can just add a bit like:

```
rule foo:
    intput:
        input.txt
    output:
        output.txt
    threads:
         8
    run:
         shell('echo {threads}')
```